gslib/commands/notification.py

```diff
@@ -561,7 +561,7 @@ def _Create(self):
             raise CommandException(
                 'Custom attributes specified with -m should be of the form '
                 'key:value')
-          key, value = a.split(':')
+          key, value = a.split(':', 1)
```

Comment:
Would str.partition be better for a single split?

Reply:
I guess either should be fine? I will leave it as it is for now.